src/main/resources/react4xp/_entries/Popup.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { X, Clipboard } from 'react-feather'
+
+const Popup = () => {
+  const [isOpen, setIsOpen] = useState<boolean | null>(null)
+  const [isVisible, setIsVisible] = useState(false)
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 767)
+  const [isScrolled, setIsScrolled] = useState(false)
+  const [hasUserScrolled, setHasUserScrolled] = useState(false)
+  const popupContainerRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    const initialIsMobile = window.innerWidth <= 767
```

Comment:
Trens denne når den settes i state for isMobile?

Reply:
Removed useEffect